shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/repositories/TripPredictionsRepository.kt

```diff
@@ -27,6 +30,10 @@ class TripPredictionsRepository(private val socket: PhoenixSocket) :
 
     override var lastUpdated: Instant? = null
 
+    override fun shouldForgetPredictions(predictionCount: Int) =
```

Comment:
Small thing, but I think it might be more readable to make this a boolean with a more informative variable name, then passing `predictionCount > 0` as the argument in the caller.